src/main/java/org/apache/maven/plugins/checkstyle/Violation.java

```diff
@@ -0,0 +1,153 @@
+package org.apache.maven.plugins.checkstyle;
+
+/*
```

Comment:
does copyright belong before the package statement? I'm not sure.

Reply:
Copied from `DefaultCheckstyleExecutor.java`. If wrong, that was wrong, too.

Reply:
Copied from other class

Reply:
I wouldn't assume other files are correct. They're certainly not normative. For questions like this you can refer to the style guide:

https://maven.apache.org/developers/conventions/code.html

We don't seem to mention this case there, but most other style guides do put license first; e.g. 

https://ace.apache.org/docs/coding-standards.html#beginning-comments

Reply:
Nope cannot do that. The checkstyle for this very project will complain: (header) 

`[ERROR] Violation.java:[1] (header) RegexpHeader: Zeile entspricht nicht der erwarteten Header-Zeile '^package '.`

Feel free to change all files in a separate commit.